website/src/pages/index.js

```diff
@@ -70,8 +70,8 @@ function Home() {
                 <div>
                   <Card
                     title="What is dbt?"
-                    body="dbt enables data practitioners to adopt software engineering best practices and deploy modular, reliable analytics code."
-                    link="/docs/introduction"
+                    body="dbt enables data <a href='https://www.getdbt.com'>practitioners</a> to adopt software engineering best practices and deploy modular, reliable analytics code."
+                    // link="/docs/introduction"
```

Comment:
We should give the docs team a heads up that if the link prop is set, any links within the body won't work, as clicking the card will trigger the link.

Reply:
Sounds good!! Thank you, I'll record a Loom for Mirna :)